OpenRA.Mods.Common/UtilityCommands/CheckMissingSprites.cs

```diff
@@ -38,29 +38,21 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			// any tilesets from being checked further.
 			try
 			{
-				// DefaultSequences is a dictionary of tileset: SequenceProvider
-				// so we can also use this to key our tileset checks
-				foreach (var kv in modData.DefaultSequences)
+				foreach (var kv in modData.DefaultTerrainInfo)
```

Comment:
Further to other feedback, I think this should use the deconstructing version for better readability: `foreach (var (terrainInfoKey, terrainInfo) in modData.DefaultTerrainInfo)`